packages/react-devtools/app.js

```diff
@@ -15,7 +15,10 @@ var updateNotifier = require('update-notifier');
 var pkg = require('./package.json');
 
 var mainWindow = null;
-var argv = process.argv.slice(2);
```

Comment:
I also fixed this, it called `slice(2)` twice. ([`bin.js`](https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/bin.js#L4))